Gemfile

```diff
@@ -24,31 +24,6 @@ def location_for(place, fake_version=nil)
   end
 end
 
-group :development, :unit_tests do
-  gem 'cisco_node_utils', '~> 1.0', require: false
-  gem 'rake', '~> 10.1.0',       require: false
-  gem 'rspec', '~> 3.1.0',       require: false
-  gem 'rspec-puppet',            require: false
-  gem 'mocha',                   require: false
-  gem 'puppetlabs_spec_helper',  require: false
-  gem 'puppet-lint',             require: false
-  gem 'pry',                     require: false
-  gem 'rubocop',                 require: false
-  gem 'simplecov',               require: false
-end
-
-beaker_version = ENV['BEAKER_VERSION']
-beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-group :system_tests do
```

Comment:
OK. Not sure about this as this block was added originally by PuppetLabs. I'll bring this up in today's call and get someone to take a look from their end.